freshmaker/errata.py

```diff
@@ -543,8 +552,34 @@ def has_jira_major_incidents(self, errata_id: str) -> bool:
 
         for issue in resp:
             if mi_pattern.search(issue["summary"]):
+                log.info(f"Found 'major incident' issue for advisory {errata_id}: {issue['key']}")
+                return True
+
+        return False
+
+    def has_compliance_priority_jira_label(self, errata_id) -> bool:
+        """
+        Checks if this erratas has an issue in JIRA with 'compliance_priority' label.
+
+        :param errata_id: The ID of the errata advisory
+        :type errata_id: str
+
+        :return: Wether the errata has or not a compliance priority issue in JIRA
+        :rtype: bool
+        """
+        resp = self._get_jira_issues(errata_id=errata_id)
+
+        if isinstance(resp, dict) and resp.get("error", False):
+            log.info(
+                f"Error when querying for Jira issues for advisory {errata_id}: {resp['error']}"
+            )
+            return False
+
+        for issue in resp:
+            if "compliance-priority" in issue["labels"]:
                 log.info(
-                    f"Found 'major incident' issue for advisory {errata_id}: {issue['key']} (errata_id={errata_id})"
+                    f"Found 'compliance priority' label in an issue for advisory {errata_id};",
+                    f"issue: {issue['key']}",
```

Comment:
```suggestion
                    f"Found compliance priority jira issue in advisory {errata_id}: {issue['key']}"
```

Reply:
I opted for
```python
f"Found 'compliance priority' label in issue {issue['key']} for advisory {errata_id}"
```
as I think it is a bit more clear. What do you think, @qixiang?

Reply:
Then it would be better to use the exact label name `compliance-priority` instead of `compliance priority`.